src/playbooks/with_build/install_everything.yml

```diff
@@ -22,6 +22,9 @@
 - name: Run install DNS
   import_playbook: "install_dns.yml"
 
+- name: Run install CORE
+  import_playbook: "install_core.yml"
+
 - name: Run install VCS
   import_playbook: "install_vcs.yml"
```

Comment:
This is installing the core twice.  I think this needs to be "install_vrss.yml".  We will also have to test that installing vrss after vstats is ok (that order is getting changed here)

Reply:
I did not find any problem installing vrss after vstat.

Reply:
Instead of calling vcs we are calling vrss now.